### PR TITLE
Add basic file processing for the election

### PR DESCRIPTION
**Pull request by beagleknight**
This adds the first check on the election log verifier. You can test it like this:

```bash
bin/verify test/fixtures/election-ok.tar
bin/verify test/fixtures/test/fixtures/election-wrong-chained-hashes.tar
```

In this PR I also keep building the skeleton of the verifier app, so it can change in the future to improve performance and so.

### CODE REVIEW

File: verifier/src/ballot/index.js [hunk 1]
```diff
@@ -1,8 +1,18 @@
...
+  /**
+   * Parse the audit ballot and run some verifiers.
+   *
+   * @param {String} path - an audit ballot file path.
+   * @returns {Promise<undefined>}
+   */
   verify() {
-    console.log(`Ballot is ${chalk.green("OK")}.`);
-    process.exit(0);
+    console.log(`${chalk.yellow("Verifying ballot...")}`);
```
Comment (orlera) [inline verifier/src/ballot/index.js:16]: 🎨

File: verifier/src/election/verifiers/chained-hash.verifier.js [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+    return new Promise((resolve) => {
+      readAllElectionFiles(
+        path,
+        (data, fail) => {
+          const { signed_data: signedData, chained_hash: chainedHash } = data;
+          // The first hash is computed using the election unique id.
+          if (!previousHash) {
+            const { message_id: messageId } = data;
+            const [authorityName, electionId] = messageId.split(".");
+            previousHash = [authorityName, electionId].join(".");
```
Comment (orlera) [inline verifier/src/election/verifiers/chained-hash.verifier.js:31]: I must be not seeing something somewhere, but this to me looks like
```
previousHash = messageId
```

**Review by orlera — APPROVED** [review]
Nice 🤩

File: verifier/src/election/verifiers/chained-hash.verifier.js (hunk 2 above)
Comment (beagleknight) [reply verifier/src/election/verifiers/chained-hash.verifier.js:31]: @orlera The `messageId` is longer 😅  so I am just taking the first two elements!

File: verifier/src/jwt-utils.js [hunk 3]
```diff
@@ -0,0 +1,28 @@
+/**
```
Comment (leio10) [inline verifier/src/jwt-utils.js:1]: Why don't we use a jwt library, as in the app?

**Review by leio10 — APPROVED** [review]
Nice work!! 🧶 🐈

**Review by agustibr — APPROVED** [review]

File: verifier/src/jwt-utils.js (hunk 3 above)
Comment (beagleknight) [reply verifier/src/jwt-utils.js:1]: Oh! I thought that at the moment I only need to decode it, and it is fairly simple without external libraries. When I need to check signatures I'll add it for sure 😄 !

**Pull request merged**